### DISCUSSION TITLE
질문 템플릿

### DISCUSSION THREAD

**Discussion by yeonkkk** [team-saviors/saview#2] [Announcements]
✅ 질문 템플릿 입니다. 복사해서 사용하세요.

#### 문제 상황 
- 문제 상황과 에러메세지를 작성해주세요.
```

```

#### 시도 방법 / 해결 방법
- 에러를 해결하기 위해 시도한 방법을 적어주세요.
- 에러를 해결한 상황이면 어떻게 해결했는지 공유해주세요.

#### 참고한 자료 
- 참고한 자료가 있다면 적어주세요.

**Comment by Inhoob** [comment]
굳입니다
